src/app/hellsite/components/business/TweetComposer.tsx

```diff
@@ -3,6 +3,7 @@ import clsx from "clsx";
 import { Button } from "../ui/Button";
 import { TextArea } from "../ui/TextArea";
 import { useState } from "react";
+import Avatar from "../ui/Avatar";
```

Comment:
This is entirely a matter of taste, but this is how I organize my imports:

- The first line is always React-related imports
- Then there are imports from NPM dependencies
- Then there is a blank line
- Then there are UI components imported from my project code
- Another blank line
- Library code imported from my project code
- Another blank line
- Types imported from my project code
- Another blank line
- Assets (CSS, SVGs, JSON, etc.) imported from my project code

This makes it easy to scan the list of imports and figure out what's being imported at a glance. But this isn't really a rule, just something I like to do in my own code.

Some people also like to sort their imports alphabetically. I don't do that, but I also don't mind doing that if it can be automated somehow.